scripts/wsmux.sh

```diff
@@ -95,13 +159,18 @@ wsmux() {
 	done
 
 	if [[ ! -z "$SELECTION" && "$SELECTION" != "false" ]]; then
-			source $SELECTION
-			export CATKIN_DIR
+			source $SELECTION/devel/setup.sh
+			export CATKIN_DIR=$SELECTION
 	fi
 
 	unset COMPREPLY
 }
 
+# Generates the configuration file if it does not exist
+if [[ ! -f $WS_CONFIG_FILE ]]; then
+	echo "WORKSPACE_BINDINGS=(	" > $WS_CONFIG_FILE
```

Comment:
This could be done in one line, but it would get very long as more and more users bind their sessions. I figured keeping one binding per line would be more effective formatting.